sklearn/sklearn-template/template/scripts/train.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+# Copyright 2018 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Convenience script for running ML training jobs.
+#
+# Prerequisites:
+#   - Google Cloud SDK
+#
+# Globals:
+#   PROJECT_ID: Google Cloud project to use.
+#   BUCKET_ID: Google Cloud Storage bucket to store output.
+#
+# Arguments:
+#  $1: Path or BigQuery table to dataset for ML training and eval
+#  $2: Whether to run `local` (on-prem) or `remote` (GCP)
+
+
+INPUT=$1
+RUN_ENV=$2
+
+if [[ ! "$RUN_ENV" =~ ^(local|remote)$ ]]; then
+  RUN_ENV=local;
+fi
+
+NOW="$(date +"%Y%m%d_%H%M%S")"
+PROJECT_NAME="sklearn_template"
+
+JOB_NAME="${PROJECT_NAME}_training_${NOW}"
+JOB_DIR="gs://$BUCKET_ID/$JOB_NAME"
+PACKAGE_PATH=trainer
+MAIN_TRAINER_MODULE=$PACKAGE_PATH.task
+
+# TODO(cezequiel): Add support for hp tuning config
+CONFIG_FILE=config.yaml
+
+if [ "$RUN_ENV" = 'remote' ]; then
+  RUN_ENV_ARGS="jobs submit training $JOB_NAME \
+    --region us-central1
+    --runtime-version 1.13
+    --python-version 2.7
```

Comment:
Can add a comment to emphasize that we also support python3

Reply:
Moved these params to `config.yaml`. Add comment to that file.

Reply:
Just added the `config.yaml` (sorry forgot to include it in original commit).